### ISSUE TITLE
Crash 1.15.2

### ISSUE THREAD

**Issue by PanoptesDreams** [dexman545/Iguana-in-a-Blanket#4]
Hard crash before launch.

Crash report:
https://pastebin.com/REqEjGN0

Log:
https://pastebin.com/CdugCS2B

**Comment by Desempregago** [comment]
Also happens in the 1.16 version (1.0.1, both snapshots)

[Crash Report](https://pastebin.com/r2fy55Gi)

**Comment by dexman545** [comment]
Is this a particular modpack you're using?

**Comment by PanoptesDreams** [comment]
It is a modpack I've been working on.

**Comment by dexman545** [comment]
Does it happen with just iguana active? I believe there is some mod conflict going on as I am unable to reproduce this.

**Comment by PanoptesDreams** [comment]
Iguana (edit: or the conflicting mod) is causing the crash, it's definitely a conflict. The other mods I'm using are in the crash report

**Comment by Desempregago** [comment]
in my case it doesn't seem to be a conflict, because I don't have any mods that edit the inventory apart from sorting

**Comment by dexman545** [comment]
I am unable to reproduce this. If you could try with just iguana installed, or narrow down the potential conflict list that would be great

**Comment by PanoptesDreams** [comment]
Yeah I've been working on it but I'm at uni so not much time, I'll let you know if I crack it
Edit: I have a lot of mods...

**Comment by Desempregago** [comment]
> I am unable to reproduce this. If you could try with just iguana installed, or narrow down the potential conflict list that would be great

Found it, it was `Not Enough Crashes` that was causing the crash
![image](https://user-images.githubusercontent.com/14002834/75101472-d17b8580-55bb-11ea-8b48-d54f3ff41d69.png)

**Comment by Desempregago** [comment]
OP, if your pack has it, you'll need to remove

**Comment by PanoptesDreams** [comment]
I don't use not enough crashes, it coincidentally does its namesake. It crashes the game too much so I haven't used it in a long time.

**Comment by PanoptesDreams** [comment]
@Fogrits you need to make your own issue that way it's tracked separately
We're not even on the same version

**Comment by PanoptesDreams** [comment]
@dexman545  I have made some progress and whittled it down to a shortlist, I'll let you know when I find the offending mod(s)

**Comment by dexman545** [comment]
I was able to reproduce the crash on 1.15.2 with NEC.

**Comment by PanoptesDreams** [comment]
> I was able to reproduce the crash on 1.15.2 with NEC.

NEC is not the offending mod, that is a seperate issue to that one, can someone make a separate issue

**Comment by dexman545** [comment]
@PanoptesDreams try disabling bunch o' trinkets, another user reported that as the cause.

**Comment by PanoptesDreams** [comment]
> @PanoptesDreams try disabling bunch o' trinkets, another user reported that as the cause.

Bunch o trinkets is on my short list of potential mods causing the issue

**Comment by PanoptesDreams** [comment]
Can confirm it is bunch o trinkets.
Sorry its taken so long

**Comment by dexman545** [comment]
Alight.
To provide a bit of perspective on what's going on:
Iguana uses a library called Item Inventory to look inside shulker boxes and other modded items that have inventories. Bunch o' Trinkets also uses this library for the same reason (this isn't a problem since Fabric Loader is meant to handle multiple of the same dependency). For some reason, the code it is looking for cannot be found, so it crashes (well, another library crashes).
(Not Enough Crashes is some strangeness that is also causing this).
I have let the author of Item Inventory know about this, but for the time being a fix being found is up in the air as neither of us has any idea _why_ it's happening.

**Comment by PanoptesDreams** [comment]
Awesome, thanks for the update.
I love knowing whats going on, on the inside. Also helps when tracking other conflicts

**Comment by dexman545** [comment]
This will fix the compatibility issues. It's an ugly, manual fix so it won't be going on curse till I have something more permanent. Rename the file from .zip to .jar and use this.
[iguana-blanket-1.0.2.zip](https://github.com/dexman545/Iguana-in-a-Blanket/files/4290070/iguana-blanket-1.0.2.zip)

**Comment by PanoptesDreams** [comment]
Hey hey, it works!
